android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewItem.kt

```diff
@@ -0,0 +1,40 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Crewman
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.utils.widget.MockView
+
+
+@Composable
+fun CrewItem(
+    crewman: Crewman,
+    modifier: Modifier = Modifier
+){
+    Column {
```

Comment:
pass modifier to top level component (Column). Currently modifier parameter isn't used. 
The idea behind modifier parameter is that we want our components to be as generic as possible and then we can pass things like size, padding, background and so on when we call them  based on the needs of the screen.